src/t8_cmesh/t8_cmesh_trees.cxx

```diff
@@ -704,10 +783,27 @@ t8_cmesh_trees_add_ghost_attribute (const t8_cmesh_trees_t trees, const int proc
   attr_info->package_id = attr->package_id;
   attr_info->attribute_size = attr->attr_size;
 
+#if 0
```

Comment:
What is this dead code doing here? Please either delete it or turn it into a code segment for debug mode.